Protos/V1/agent_common.proto

```diff
@@ -70,13 +73,144 @@ message Result {
     InitKeyedDataStream init = 1;
     DataChunk data = 2;
   }
-  string communication_token = 3;
+  string communication_token = 3; /** Communication token received by the worker during task processing */
 }
 
 message ResultReply {
-  string communication_token = 3;
+  string communication_token = 3; /** Communication token received by the worker during task processing */
   oneof type {
     Empty Ok = 1;
     string Error = 2;
   }
 }
+
+/*
+* Request for creating results without data
+*/
+message CreateResultsMetaDataRequest {
+  /**
+  * A result to create.
+  */
+  message ResultCreate {
+    string name = 1; /** The result name. Given by the client. */
+  }
+  repeated ResultCreate results = 1; /** The list of results to create. */
+  string session_id = 2; /** The session in which create results. */
+  string communication_token = 3; /** Communication token received by the worker during task processing */
+}
+
+/**
+* Result metadata
+*/
+message ResultMetaData {
+  string session_id = 1; /** The session ID. */
+  string result_id = 2; /** The result ID. */
+  string name = 3; /** The result name. */
+  result_status.ResultStatus status = 4; /** The result status. */
+  google.protobuf.Timestamp created_at = 5; /** The result creation date. */
```

Comment:
I think we could need an end time to have the time when the result has been completed or turn into an error. Could be a later addition.

Reply:
Let's create some issues before resolving this comment